packages/components/src/toolbar/style.scss

```diff
@@ -72,12 +72,12 @@
 		}
 
 		// Ensure the icon buttons remain square.
-		&.has-icon {
+		// This needs specificity.
+		&.has-icon.has-icon {
```

Comment:
What's the double class for?

Reply:
Short answer: it incrementally increases the specificity of this rule. 

Longer answer, it adds specificity to override this inherited padding:

<img width="389" alt="Screenshot 2021-06-28 at 14 29 56" src="https://user-images.githubusercontent.com/1204802/123636609-5d8b6600-d81d-11eb-80eb-9e46e0c422dd.png">

This wasn't necessary in the past, since the icon was centered inside the 48x48 button. But once you add text, such as in reusable blocks, that lower padding will be leveraged. By setting higher specificity and bigger padding, we get it right:

<img width="281" alt="Screenshot 2021-06-28 at 14 30 46" src="https://user-images.githubusercontent.com/1204802/123636827-9b888a00-d81d-11eb-979d-1d31ee170daf.png">

Much longer answer: the block toolbar works decently as is. But having grown organically from features across the years, the CSS is at a level of complexity where it'd be nice to revisit it in a refactor.